roles/kubernetes-apps/metallb/templates/metallb.yaml.j2

```diff
@@ -1724,6 +1724,9 @@ spec:
       - args:
         - --port={{ metallb_port }}
         - --log-level={{ metallb_log_level }}
+{% if 'loadbalancer_class' in metallb_config.keys() and metallb_config.loadbalancer_class != "" %}
+        - --lb-class={{ metallb_config.loadbalancer_class }}
+{% endif %}
```

Comment:
Thanks @Seal1998 

Is is better to define the loadbalancer_class  at the `https://github.com/kubernetes-sigs/kubespray/blob/v2.23.0/roles/kubernetes-apps/metallb/defaults/main.yml#L4` instead of the `metallb_config` the same as the `metallb_port `

Reply:
Hey @yankay, thanks for review! I have changed the condition and role defaults accordingly. Unfortunately, to my understanding, we cannot have the `--lb-class` flag defined by default, similar to `--port`. This is because it would instruct MetalLB to listen for a specific class of LoadBalancer. However, Kubernetes does not have any "default" class to listen for, as per its documentation. The loadBalancerClass is undefined by default on all LoadBalancer services. (https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class)

> By default, .spec.loadBalancerClass is not set and a LoadBalancer type of Service uses the cloud provider's default load balancer implementation if the cluster is configured with a cloud provider using the --cloud-provider component flag.

In my opinion, we should keep `--lb-class` undefined until the user specifically sets the `metallb_loadbalancer_class` variable to something meaningful. In this way, MetalLB will act as the 'default' LoadBalancer implementation until we specifically set `--lb-class`

Official metallb [helmchart](https://github.com/metallb/metallb/blob/4f7e29972652a6f415ad42898d0b5ef5adc06440/charts/metallb/templates/controller.yaml#L69C9-L71C19) also uses this approach
```
{{- if .Values.loadBalancerClass }}
        - --lb-class={{ .Values.loadBalancerClass }}
{{- end }}
```
Please review